cmd/frontend/backend/app.go

```diff
@@ -0,0 +1,97 @@
+package backend
+
+import (
+	"context"
+	"github.com/sourcegraph/sourcegraph/internal/database"
+	"github.com/sourcegraph/sourcegraph/internal/extsvc"
+	"github.com/sourcegraph/sourcegraph/internal/jsonc"
+	"github.com/sourcegraph/sourcegraph/internal/types"
+	"github.com/sourcegraph/sourcegraph/lib/errors"
+	"github.com/sourcegraph/sourcegraph/schema"
+)
+
```

Comment:
nit: you can also add a type guard just in case.

```suggestion

var _ AppExternalServicesService = &appExternalServices{}

```